packages/material-ui-styles/src/withTheme/withTheme.d.ts

```diff
@@ -13,6 +13,7 @@ export function withThemeCreator<Theme = DefaultTheme>(
 export interface WithTheme<Theme = DefaultTheme> {
   theme: Theme;
   innerRef?: React.Ref<any>;
+  ref?: React.Ref<any>;
```

Comment:
Since this is new and we do rely on ^3.0 I'd rather use `unknown` here. (we probably need to widen this later but let's not start out with legacy any.)